src/test/java/org/jenkinsci/plugins/docker/workflow/DockerDSLTest.java

```diff
@@ -333,7 +333,7 @@ private static void grep(File dir, String text, String prefix, Set<String> match
                 p.setDefinition(new CpsFlowDefinition(
                         "node {\n" +
                                 "     def busybox = docker.image('busybox');\n" +
-                                "     busybox.run('--rm', 'echo \"Hello\"');\n" +
+                                "     busybox.run('--tty', 'echo \"Hello\"').stop();\n" +
```

Comment:
🐛 I think - is the container deleted elsewhere?  If not, we still need the --rm or the old containers will pile up.